packages/react-bootstrap-table2/test/mock-data/body-resolved-props.js

```diff
@@ -0,0 +1,16 @@
+import Const from '../../src/const';
```

Comment:
Hi @AllenFang 

I created `body-resolved-props.js` and `header-resolved-props.js` to allow developers to mock the props which must exist like `cellEdit` or `rowSelect` much easier. If you have any suggestions plz let me know :)

Reply:
it's good, but you can write it more smart.

write some factory function to generate the resolved props and this function allow this function pass some arguments like `data` and `column` and `keyField`. Also, in the future, if in the test, we need some custom `cellEdit`, we can change this function more easier.

Reply:
Sure I'll enhance it in the future.